lib/franklin_web/controllers/syndication_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule FranklinWeb.SyndicationController do
+  use FranklinWeb, :controller
+
+  alias Franklin.Articles
+  alias FranklinWeb.RssFeed
+
+  @doc """
+  GET /index.xml
+
+  Returns an RSS feed for the most recent articles.
+  """
```

Comment:
Is it a good idea for the inline doc of a controller function to express the expected path it will be called from?

Feels a little bad to couple and make assumptions about what the Router is doing, but 🤷‍♂️